doc/documentation/caching.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Caching
+description:
+    Custom response caching for greater application flexibility.
+---
+
+
+When building a complex web application, sometimes you want more
+flexibility than standard HTTP based caching in order to best
+balance your application logic with performance.  SPF provides
+a local configurable response cache that you can adjust as
+needed.
+
+
+## When the cache is used
+
+Before making a request, SPF checks the local cache to see if a
+valid cached response is available for the requested URL.  If
+one is found, then it will be used instead of sending a request
+over the network.  If not, after the response is received, SPF
+will place that response in the local cache for future use.
+
+By default, SPF uses a caching model that matches the back-
+forward cache in browsers: "new" navigation will not use a
+previously cached response, whereas "history" navigation will.
+This means that your server will receive requests for every link
+click, for example, but not for back button uses.  However, if
+the `cache-unified` config value is set to `true`, then SPF will
+use an available cache response for all navigation.  This will
+mean that your server will not receive requests for link clicks
+that are to previously visited pages, just like for back button
+presses.
+
+When [prefetching][] responses, by default, SPF will place a
+prefetched response in the local cache as eligible for one "new"
+navigation.  After that one use, the cached response will only
+be eligible for "history" navigation.  However, if the
+`cache-unified` config value is set to `true`, then the
+prefetched response will be available for all navigation like
+other cached responses.
+
+
+## Configuring the cache
+
+You may configure the cache using parameters to control the
+total number and lifetime of entries in the cache.  Increasing
+these settings will increase the chance of finding a valid
```

Comment:
valid cached -> valid, cached

I think you should have a comma here. Not really an expert though.